xbmc/Application.cpp

```diff
@@ -211,7 +211,7 @@
 #if defined(TARGET_ANDROID)
 #include "platform/android/activity/XBMCApp.h"
 #include "platform/android/activity/AndroidFeatures.h"
-#include "platform/android/jni/Build.h"
+#include "androidjni/Build.h"
```

Comment:
maybe the order of the includes for the androidjni files can also be adjusted to our [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html#code_guidelines_1_3)